WSL/toc.yml

```diff
@@ -1,43 +1,37 @@
+- name: WSL Documentation
+  href: index.md
 - name: Overview
-  href: overview.md
```

Comment:
We removed overview.md . Not sure if that's on purpose or if we do want it anywhere.

Reply:
On purpose